app/views/templates/_form.html.erb

```diff
@@ -9,33 +9,26 @@
   <ul class="nav nav-tabs" data-tabs="tabs">
     <li class="active"><a id="primary_tab" href="#primary" data-toggle="tab"><%= _("Template") %></a></li>
     <%= render "#{@type_name_plural}/custom_tab_headers" unless type == 'ptable'%>
-
     <li><a id='history_tab' href="#history" data-toggle="tab"><%= _("History") %></a></li>
     <% if show_location_tab? %>
       <li><a href="#locations" data-toggle="tab"><%= _("Locations") %></a></li>
     <% end %>
     <% if show_organization_tab? %>
       <li><a href="#organizations" data-toggle="tab"><%= _("Organizations") %></a></li>
     <% end %>
+    <% if Setting[:safemode_render] %>
```

Comment:
> This will need to be always displayed when moving the link to docs.

In the previous comment, I'd suggested moving the link to the useful macros etc to the Help tab, which you said you'd move. If it moved then the help tab would always need to be visible - which it now is, so sorted.